assets/js/modules/pagespeed-insights/settings/settings-view.js

```diff
@@ -26,11 +26,11 @@ import { __, sprintf } from '@wordpress/i18n';
  */
 import Data from 'googlesitekit-data';
 import { sanitizeHTML } from '../../../util';
-import { STORE_NAME as CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME } from '../datastore/constants';
 const { useSelect } = Data;
 
 export default function SettingsView() {
-	const dashboardPermalink = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard' ) );
+	const dashboardPermalink = useSelect( ( select ) => select( STORE_NAME ).getAdminReauthURL() );
```

Comment:
This should not use `getAdminReauthURL`, but `getAdminScreenURL` - it's just for viewing the data, not for setting up the module.

Reply:
Done